Cargo.toml

```diff
@@ -12,3 +12,8 @@ serde = { version = "1.0.140", features = ["derive"] }
 serde_json = "1.0.82"
 uuid = { version = "1.1.2", features = ["v4", "serde"] }
 derive_builder = "0.11.2"
+fake = { version = "2.5.0", features = ["derive", "uuid"] }
+rand = "0.8"
```

Comment:
Is this really used? Just searching for it in the project didn't show me any matches.

Reply:
`fake` (and `rand`) have been removed for now